### DISCUSSION TITLE
Being able to specify a color in hexadecimal

### DISCUSSION THREAD

**Discussion by kud** [easonwong-de/Adaptive-Tab-Bar-Colour#94] [Ideas]
Hey hey,

It could be great to be able to specify a colour in hexadecimal, it'll be faster than the colour picker.

Thanks!

**Comment by easonwong-de** [comment]
Hi, @kud! No worries, the new colour picker will be way better. Here's the preview:

![image](https://github.com/user-attachments/assets/f206c803-61e5-4c53-b714-5713e6d76bd5) 

Thanks for the suggestion!

**Reply by kud** [reply]
Oh god, that's so neat! Thank you 😊
